docs/output.md

```diff
@@ -163,6 +163,31 @@ The summary file lists all statistics of kraken2 and/or bbduk (and optionally bl
 
 Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQC. The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability. For more information about how to use MultiQC reports, see <http://multiqc.info>.
 
+### Downstream samplesheets
+
+The pipeline can also generate input files for the following downstream
+pipelines:
+
+- [nf-core/taxprofiler](https://nf-co.re/taxprofiler)
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `downstream_samplesheets/`
+  - `taxprofiler.csv`: Filled out nf-core/taxprofiler `--input` csv with paths to reads relative to the results directory
+  - `mag.csv`: Filled out nf-core/mag `--input` csv with paths to reads relative to the results directory
+
+</details>
+
+:::warning
+Any generated downstream samplesheet is provided as 'best effort' and are not guaranteed to work straight out of the box!
+They may not be complete (e.g. some columns may need to be manually filled in).
+:::
+
+:::warning
+Detaxizer can process long-reads independent from short reads. `nf-core/mag@v3.1.0` can only take short, or short+long but not standalone long-reads as an input (this is being worked on). If you want to use the output of Detaxizer for Mag, you'll have to remove the standalone long reads from the `mag.csv` file.
```

Comment:
```suggestion
nf-core/detaxizer can process long-reads independent from short reads, however ll versions of  nf-core/mag up until v3.1.0 can only take short, or short+long but not standalone long-reads as an input.

Long-read only data will be removed
```

I think we should just not include the long reads (only) lines at all until supported in mag.